src/device/usbd.c

```diff
@@ -45,14 +45,15 @@ typedef struct {
   {
     volatile uint8_t connected    : 1;
     volatile uint8_t addressed    : 1;
-    volatile uint8_t configured   : 1;
     volatile uint8_t suspended    : 1;
 
     uint8_t remote_wakeup_en      : 1; // enable/disable by host
     uint8_t remote_wakeup_support : 1; // configuration descriptor's attribute
     uint8_t self_powered          : 1; // configuration descriptor's attribute
   };
 
+  volatile uint8_t selected_config;
```

Comment:
can we rename it to have config_xxx, maybe config_value or config_num or config_selected . I kind of like the most relevant word appear first scheme.

Reply:
Oh, the irony.  I had "config_num" in my local code, but then changed it to be the same as #167.